README.md

```diff
@@ -65,6 +63,206 @@ See [Amplication website](http://amplication.com/) or [Amplication docs](http://
 
 Launch Amplication from [app.amplication.com](http://app.amplication.com/)
 
+# System Requirements
+
+## Amplication Server
+
+Amplication Server is the main component of the platform that provides all the core functionality to design and create low-code applications.
+The server exposes a GraphQL API for all actions. The server is built with the following awesome open source technologies: Node.js, NestJS, Prisma over PostgreSQL, GraphQL API, and many more...
+
+If you need help or have a question, go to our [Discord channel](https://discord.gg/Z2CG3rUFnu), we are here to help.
+
+### Development
+
+:bulb: Before you begin, make sure you have all the below installed:
+
+- [Node.js v14 or above](https://nodejs.org/en/download/)
+- [npm v7 or above](https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/)
+- [Docker](https://docs.docker.com/desktop/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git/)
+
+#### Automatic one-time setup
+
+Amplication is using a mono-repo with multiple packages. To initialize all the packages on a local development environment, you should follow the instruction on the [README.md](../../README.md) file in the project root folder.
+
+You can also use a more manual step-by-step approach to set up Amplication server - to do that, follow the instructions below.
+
+#### Manual one-time set up
+
+- Install dependencies of the monorepo (execute in root directory):
+  ```
```

Comment:
@GaneshSonarSD - please make this a numbered list. We use bullets only when it doesn't matter in which order the steps are performed.  Automatic numbering is sometimes problematic in Markdown. I suggest you type in the numbers manually.